### ISSUE TITLE
can not edit the inference labels for the radilogy app

### ISSUE THREAD

**Issue by luxunxiansheng** [Project-MONAI/tutorials#1605]
I follow the tutorial  monailabel_radiology_spleen_segmentation_OHIF.ipynb   and can start the app.  When I want to edit the inference labels as shown below:
![image](https://github.com/Project-MONAI/tutorials/assets/26159816/d5d63acd-c0ec-4257-acd8-922ee968935f)

I can not find the "segmentation" tool buttons and no upleft  pannel either.   

The info looks like this:

`{
    "name": "MONAILabel - Radiology (0.8.1)",
    "description": "DeepLearning models for radiology",
    "version": "0.8.1",
    "labels": [
        "spleen"
    ],
    "models": {
        "segmentation_spleen": {
            "type": "segmentation",
            "labels": {
                "spleen": 1
            },
            "dimension": 3,
            "description": "A pre-trained model for volumetric (3D) segmentation of the spleen from CT image",
            "config": {
                "device": [
                    "NVIDIA GeForce RTX 3090:0",
                    "NVIDIA GeForce RTX 3090:1",
                    "NVIDIA GeForce RTX 3090:2",
                    "NVIDIA GeForce RTX 3090:3"
                ]
            }
        },
        "Histogram+GraphCut": {
            "type": "scribbles",
            "labels": {
                "spleen": 1
            },
            "dimension": 3,
            "description": "A post processing step with histogram-based GraphCut for Generic segmentation",
            "config": {
                "num_bins": 64,
                "lamda": 1,
                "sigma": 0.1,
                "device": [
                    "NVIDIA GeForce RTX 3090:0",
                    "NVIDIA GeForce RTX 3090:1",
                    "NVIDIA GeForce RTX 3090:2",
                    "NVIDIA GeForce RTX 3090:3"
                ]
            }
        },
        "GMM+GraphCut": {
            "type": "scribbles",
            "labels": {
                "spleen": 1
            },
            "dimension": 3,
            "description": "A post processing step with GMM-based GraphCut for Generic segmentation",
            "config": {
                "num_mixtures": 20,
                "lamda": 5,
                "sigma": 0.5,
                "device": [
                    "NVIDIA GeForce RTX 3090:0",
                    "NVIDIA GeForce RTX 3090:1",
                    "NVIDIA GeForce RTX 3090:2",
                    "NVIDIA GeForce RTX 3090:3"
                ]
            }
        }
    },
    "trainers": {
        "segmentation_spleen": {
            "description": "Train Spleen Segmentation Model",
            "config": {
                "name": "train_01",
                "pretrained": true,
                "device": [
                    "NVIDIA GeForce RTX 3090:0",
                    "NVIDIA GeForce RTX 3090:1",
                    "NVIDIA GeForce RTX 3090:2",
                    "NVIDIA GeForce RTX 3090:3"
                ],
                "max_epochs": 50,
                "early_stop_patience": -1,
                "val_split": 0.2,
                "train_batch_size": 1,
                "val_batch_size": 1,
                "multi_gpu": true,
                "gpus": "all",
                "dataset": [
                    "SmartCacheDataset",
                    "CacheDataset",
                    "PersistentDataset",
                    "Dataset"
                ],
                "dataloader": [
                    "ThreadDataLoader",
                    "DataLoader"
                ],
                "tracking": [
                    "mlflow",
                    "None"
                ],
                "tracking_uri": "",
                "tracking_experiment_name": ""
            },
            "labels": {
                "spleen": 1
            }
        }
    },
    "strategies": {
        "random": {
            "description": "Random Strategy"
        },
        "first": {
            "description": "Get First Sample"
        },
        "last": {
            "description": "Get Last Sample"
        }
    },
    "scoring": {},
    "train_stats": {
        "segmentation_spleen": {}
    },
    "datastore": {
        "total": 1,
        "completed": 0,
        "label_tags": {}
    }
}`

**Comment by ericspod** [comment]
@tangy would be able to help here? Thanks!

**Comment by tangy5** [comment]
Hi @luxunxiansheng , the MONAI plugin does not provide any label editing functions. The annotation funcs are within OHIF. The tutorial used OHIF V2 . The brushes and erases should show as screenshot. The reason could be the OHIF compile. 
Did you observe any errors when build the OHIF or start OHIF?

**Comment by luxunxiansheng** [comment]
> Hi @luxunxiansheng , the MONAI plugin does not provide any label editing functions. The annotation funcs are within OHIF. The tutorial used OHIF V2 . The brushes and erases should show as screenshot. The reason could be the OHIF compile. Did you observe any errors when build the OHIF or start OHIF?

here is the screenshot:

![image](https://github.com/Project-MONAI/tutorials/assets/26159816/7b2a610a-fcf5-459a-bb01-24ae630b57ff)

there seem no errors when I start OHIF.

**Comment by tangy5** [comment]
This looks strange to me (As OHIF conpile problem). And there is no "segmentation" dropdown panel when you hit the "+" on the top right? If so, I will give a try and investigate more, then update all necessary docs.

**Comment by diazandr3s** [comment]
Hi @luxunxiansheng,

It seems you installed the latest MONAI Label version (0.8.1) which uses the latest OHIF version (V3). 

The tutorial [monailabel_radiology_spleen_segmentation_OHIF.ipynb](https://github.com/Project-MONAI/tutorials/blob/main/monailabel/monailabel_radiology_spleen_segmentation_OHIF.ipynb) uses a previous version of MONAI Label. 

This has to be updated in the tutorial documentation.

For now, I'd suggest you install a previous version of MONAI Label that uses OHIFV2. i.e. pip install monailabel==0.7.0

Let us know,

**Comment by luxunxiansheng** [comment]
OK. I will install 0.7.
